devops/create_https_wss_endpoints_aws/main.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/env python3
+
+'''
+
+PURPOSE:
+
+create https/wss endpoints for a network
```

Comment:
Is this scrip always used to create end points from scratch?  Can it be used to update existing end points, such like we launched testnet multiple times? If yes, how?  Please add it in the usage.

Reply:
yes, it is described in the __main__ func.